config/initializers/carrierwave.rb

```diff
@@ -3,12 +3,12 @@
     config.fog_provider = 'fog/aws'
     config.fog_credentials = {
       provider:              'AWS',
-      aws_access_key_id:     ENV['S3_KEY'],
-      aws_secret_access_key: ENV['S3_SECRET'],
-      region:                ENV['S3_REGION'],
+      aws_access_key_id:     Rails.application.credentials.dig(:aws, :access_key_id),
```

Comment:
```suggestion
      aws_access_key_id:     Rails.application.credentials.aws.access_key_id
```

Let it throw error if not found because the values are required for the code to work.